### ISSUE TITLE
AbbreviatedFieldName false positive on property defined in base class

### ISSUE THREAD

**Issue by ccecvb** [Riverside-Software/sonar-openedge#1224]
I get Abbreviated field name for STATEMENT_OR_ROW on `if State`
```
class be.cce.ccetools.mq.StompMQClient
    inherits BaseMQClient:

destructor public StompMQClient( ):
if State = {&MQState_Connected}
then this-object:Disconnect().
  
end destructor.
```

State is a property defined in BaseMQClient

**Comment by gquerret** [comment]
This only happens if the typeInfo structure (ie rcode) of BaseMQClient is not available in proparse. Make sure sonar-scanner or sonarlint has the correct propath.
